packages/google_sign_in/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 0.2.1
```

Comment:
@collinjackson Is the codelab locked to a plugin version? Do I need to update anything in the codelab regarding https://github.com/flutter/flutter/issues/10688?